LeadKit/LeadKit/Extensions/Notification/Notification+UIKeyboardInfoValues.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Notification+UIKeyboardInfoValues.swift
+//  LeadKit
+//
+//  Created by Ivan Smolin on 12/12/16.
+//  Copyright © 2016 Touch Instinct. All rights reserved.
+//
+
+import Foundation
+
+public extension Notification {
+
+    /// CGRect value for UIKeyboardFrameBeginUserInfoKey or nil
+    var keyboardFrameBegin: CGRect? {
+        return (userInfo?[UIKeyboardFrameBeginUserInfoKey] as? NSValue)?.cgRectValue
+    }
```

Comment:
did you mean `KeyboardDidNotificationValues` and `KeyboardWillNotificationValues`?